react/MonthPicker/MonthPicker.js

```diff
@@ -113,13 +113,17 @@ export default class MonthPicker extends Component {
 
     return (
       <div className={classNames}>
-        <FieldLabel
-          {...{ id, label, labelProps, secondaryLabel, tertiaryLabel }}
-        />
-        {this.renderInput()}
-        <FieldMessage
-          {...{ invalid, help, helpProps, valid, message, messageProps }}
-        />
+        <fieldset>
```

Comment:
Rather than nest the `fieldset` in the `div` you can change the `div` to a `fieldset` for the same result right?

Reply:
Having a re-think about this one I'm not a fan as nested fieldset has problems, I'll close for now and work on a better solution.